src/Components/Movies.js

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+const Movies = () => {
+  const [movies, setMovies] = useState([]);
+  const [chosenMovie, setChosenMovie] = useState("");
+  
+
+  const fetchMovies = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+      setMovies(res.data);
+    } catch (error) {
+      setMovies([]);
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    fetchMovies();
+  }, []);
+
+  const handleChange = async (e) => {
+    setChosenMovie(e.target.value);
```

Comment:
I'm not sure why you're calling this function to set state here and below. You're changing the sate twice in the same function so it will really only do last one